conf/kvs-config.yml

```diff
@@ -24,10 +24,12 @@ ebs: ./
 threads:
   memory: 4
   ebs: 4
+  sharedmemory: 4
```

Comment:
Nit: Throughout this should be `shared-memory` or `shared_memory`. Or maybe we want to call this the `immutable` tier or something?